share/dbtypes_gen.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+
+with open(os.path.join(os.path.dirname(__file__), '..', 'src', 'query_backend.h')) as f:
+    lines = f.readlines()
+
+VERSION = "v"+sys.argv[1]
+
+ALL_BACK = ["SQLite", "HDF5"]
```

Comment:
This is a container and so it is conventional fore the variable name to be plural, ie `ALL_BACKS` or maybe `BACKS` or `BACKENDS`